src/pages/Login.jsx

```diff
@@ -0,0 +1,75 @@
+import { Link } from "react-router-dom";
+import authbg from "../assets/auth_bg.png";
+
+const Login = () => {
+     return (
+          <>
+               <div className="h-screen flex flex-row">
+                    <div className="h-full w-full md:w-1/2 flex justify-center items-center">
+                         <div className="h-fit w-full md:w-[450px] flex flex-col justify-center gap-5 md:gap-10">
+                              <h1 className="font-normal text-md md:text-lg text-white font-Bayon text-center">Welcome to <span className="text-primary-default">Will Be There</span></h1>
+                              <form noValidate className="flex flex-col gap-5 md:gap-10">
+                                   <div className="px-5 flex flex-col gap-5 md:gap-10">
+                                        {/* <div className="flex flex-col gap-10"> */}
+                                        <div className="flex flex-col font-Bayon">
+                                             <h2 className="text-base md:text-md text-start uppercase text-primary-default">login</h2>
+                                             <p className="text-slate uppercase font-Bayon font-normal text-sm md:text-base">doesn’t have an account? <Link to="/signup" className="text-primary-default">sign up</Link></p>
+                                        </div>
+                                        {/* inputs */}
+                                        <div className="w-full flex flex-col font-Bayon gap-5 md:gap-[37px]">
+                                             <input type="text" name="first_name" required autoComplete="none" placeholder="enter user name"
```

Comment:
Make the input element a reusable component. Also, having both "user name" and "email field" for login is redundant. One is enough, preferably the email field.